plugins/modules/zos_copy.py

```diff
@@ -2454,6 +2485,34 @@ def run_module(module, arg_def):
         )
 
     # ********************************************************************
+    # To validate the source and dest are not lock in a batch process by
+    # the machine and not generate a false positive check the disposition
+    # for try to write in dest and if both src and dest are in lock.
+    # ********************************************************************
+    if src_ds_type != "USS" and dest_ds_type != "USS":
+        is_source_lock = source_locked(src_name)
+        is_dest_lock = source_locked(dest_name)
+        if not is_source_lock and not is_dest_lock:
+            module.fail_json(
+                msg="DATASETS in lock, unable to access'{0}' without force and unable to write in'{1}'".format(
```

Comment:
I will check out the code and update the message, it does get the point out to the user but we can be a bit more grammatically correct. @jennyhuang17 fyi

Reply:
Thanks for the heads up @ddimatos . I agree we should be careful with the accuracy and the readability of messages that are going to be seen be users. Although it would be a challenge to review messages in codes, let's figure out a way for this... I'll start with searching for IBM guidelines for message writing and share with the team.

Having said that, can you ask the team to invite me to review when there's doc update or message change?

Reply:
Sure Jenny, we can do that.

Reply:
Resolved